packages/util/src/defaults.ts

```diff
@@ -39,6 +39,7 @@ export interface FirebaseDefaults {
   emulatorHosts?: Record<string, string>;
   _authTokenSyncURL?: string;
   _authIdTokenMaxAge?: number;
+  forceEnvironment?: 'browser' | 'node';
```

Comment:
We should probably document every property here. If you just do `forceEnvironment` in this PR, I will go back and do the other ones in another PR later. It just needs a comment in the form of
```
/**
 * Documentation goes here
 */
```
This doesn't need an individual `@public` tag as it is covered by the one for the interface.

Reply:
Added documentation for the `forceEnvironment` property